README.md

```diff
@@ -20,44 +29,102 @@
     <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="License">
   </a>
 </p>
+ 
+## Screenshots
+
+![dontfile screenshot](https://i.imgur.com/ltqRqUl.png)
+
+## Tech/framework used
 
-### Description
+<b>Built with</b>
 
-Dontfile is an open source project intended to concatenate the proposed idea of [dontpad](http://dontpad.com/) aggregating the same logic to files along with texts, thus creating a more complete tool.
+- [Rails](https://rubyonrails.org/)
 
-* Ruby version: 2.5.1
-* Rails version: 5.2.1
+## Features
 
-### Setting up development environment
+- Active storage
+- Collaboration using URLs
+- Text saved automatically
+- File storage
 
-1. Make sure you have correct Ruby version, either with [RVM](https://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv)
+## Installation
 
-2. Install latest bundler
+### Local installation
+
+#### Prerequisites
+
+- Ruby version: 2.5.1
+- Rails version: 5.2.1
+
+#### Install latest bundler
+
+Install [bundler](https://bundler.io/)
 
 ```bash
 gem install bundler
 ```
 
-3. Bundle the necessary gems
+#### Bundle the necessary gems
 
 ```bash
 bundle install
 ```
 
-4. Create the database
+#### Create the database
 
 ```bash
 rails db:create
 ```
 
-5. Migrate the database
+#### Migrate the database
 
 ```bash
 rails db:migrate
 ```
 
-6. Run the server
+#### Run the server
 
 ```bash
 rails server
 ```
+
+### Docker
+
+#### Prerequisites for docker
+
+An installation of [Docker](https://docs.docker.com/install/)
+
+#### Steps
+
+Run docker compose
+
+```bash
+docker-compose up -d
+```
+
+Navigate to http://localhost:3000
+
+## Tests
+
+```bash
+rails test
+```
+
+## Contribute
+
+Fork and make a pull request
+
+## Credits
+
+The idea is thanks to [dontpad](http://dontpad.com)
+
+## License
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Change to:
Dontfile is under the [MIT License](LICENSE)! You can download, change, share the source code!